src/Products.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * Meant to handle special Product features.
+ */
+class Products {
+
+  const POST_TYPE_PRODUCT = 'product';
+
+  const TAX_PRODUCT_CAT = 'product_cat';
+
+  const FIELD_ENFORCE_CAT_LINKS = Plugin::PREFIX . '_enforce_main_category_links';
+
+  /**
+   * Stores the product base link.
+   *
+   * @var string
+   */
+  private string $product_base_link;
+
+  /**
+   * Initialize the product module.
+   */
+  public static function init(): void {
+    $obj = new self();
+    if (get_option(self::FIELD_ENFORCE_CAT_LINKS)) {
+      add_filter('post_type_link', [$obj, 'get_product_permalink'], 10, 2);
+    }
+    add_action('admin_init', [$obj, 'register_product_settings']);
+    add_action('admin_init', [$obj, 'save_product_settings']);
+  }
+
+  /**
+   * Registers custom product settings.
+   */
+  public function register_product_settings(): void {
```

Comment:
@chrisbaltazar why are adding settings in another place than the Shop Standards tab from WooCommerce settings?
![Uploading Screenshot 2021-12-01 at 13.16.27.png…]()